solution/ui/regulations/eregs-vite/src/views/Search.vue

```diff
@@ -351,12 +349,9 @@ export default {
         },
         async retrieveResourcesResults({ query, page, pageSize }) {
```

Comment:
I dont think we ever plan on changin this from the default of 50, can probably just remove it
```suggestion
        async retrieveResourcesResults({ query, page, pageSize }) {
```

Reply:
there were talks about eventually adding the ability for users to select how many resources they'd like to see per page.  It's also functionality the SearchGov endpoint should have if we end up using it.  I'd rather keep it here for now.

Reply:
we've set the existing Django resources API endpoint back to the default call when requesting resources.  We've also re-enabled the ability to add `searchgov` as a URL parameter to request resources from the new SearchGov endpoint.  So `pageSize` is used again.